config/zephyr/Kconfig

```diff
@@ -108,6 +108,17 @@ config CHIP_OTA_REQUESTOR
 	  Device Firmware Upgrade by quering and downloading a new firmware image
 	  from an external OTA Provider node.
 
+config CHIP_OPERATIONAL_TIME_SAVE_INTERVAL
+	int "Interval of saving node operation time to flash in hours unit"
+	default 10
+	range 1 24
```

Comment:
Not sure we should limit it to 24 hours. Is it possible to set minimum value only?